File: nexed/python-beginner/03-level/Error-Error-bace86b64621-0e4ec984c96c/calculator.py
print("Operatie?")
symbol = input("+,-,*,/, %")
try:
    getal1 = float(input("Eerste getal?"))
    getal2 = float(input("Tweede getal?"))
    if symbol == "+":
        print(getal1 + getal2)
    elif symbol == "-":
        print(getal1 - getal2)
    elif symbol == "*":
        print(getal1 * getal2)
    elif symbol == "/":
        if getal2 != 0:
            print(getal1 / getal2)
        else:
            print("Kan niet delen door 0")
    elif symbol == "%":
        if getal2 != 0:
            print(getal1 % getal2)
        else:
            print("Kan niet delen door 0")
except ValueError:
    print("Dat is geen getal")

File: nexed/php-beginner/04-level/Calculator-upgrade-73130727e3da-d5a1a161b357/calculator.php
<?php

echo "Welke operatie wil je uitvoeren? (+, -, %)";
$op = readline();

if ($op != "+" && $op != "-" && $op != "%") {
    echo "Dat is geen geldige operatie!";
    exit;
}

$getal1string = readline();
$getal2string = readline();

if (!is_numeric($getal1string) || !is_numeric($getal2string)) {
    echo "Dat is geen getal!";
    exit;
}

$getal1 = intval($getal1string);
$getal2 = intval($getal2string);

$res = 0;

if ($op == "+") {
    $res = $getal1 + $getal2;
} else if ($op == "-") {
    $res = $getal1 - $getal2;
} else if ($op == "%") {
    $res = $getal1 % $getal2;
}

echo $res;

?>